### PR TITLE
Added more commands help and bind

### PR DESCRIPTION
**Pull request by enderchickenbawk**
- added Help command

- added Bind command

### CODE REVIEW

File: src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/BindCommand.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    .build()
+            )
+            .handler { args ->
+                val name = args[0] as String
+                val module = LiquidBounce.moduleManager.find { it.name.equals(name, true) } ?: return@handler false
+                try {
+                    module.bind =
+                        GLFW::class.java.getField("GLFW_KEY_" + (args[1] as String).toUpperCase()).getInt(GLFW::class);
+                } catch(e: Exception) {
+                e.printStackTrace();
```
Comment (kawaiinekololi) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/BindCommand.kt:41]: Missing error handling on unknown keys (no user response)

File: src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/HelpCommand.kt [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import net.ccbluex.liquidbounce.features.command.builder.CommandBuilder
+import net.ccbluex.liquidbounce.utils.chat
+
+object HelpCommand {
+
+    fun createCommand(): Command {
+        return CommandBuilder
+            .begin("help")
+            .description("list commands.")
+            .handler { args ->
```
Comment (kawaiinekololi) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/command/commands/HelpCommand.kt:13]: Help command should automatically generate a list of commands with descriptions

**Review by kawaiinekololi — CHANGES_REQUESTED** [review]
read comments

**Comment by enderchickenbawk** [comment]
ok
added automatic command list generation
added "invalid keybinding." to Exception in bind command

**Comment by kawaiinekololi** [comment]
I appreciate your work, but it seems you don't really know how to write Kotlin code. I'll do it myself or might work on the pull request.

**Pull request merged**